view/theme/frio/templates/admin/users.tpl

```diff
@@ -0,0 +1,272 @@
+<script>
+	function confirm_delete(uname){
+		return confirm( "{{$confirm_delete}}".format(uname));
+	}
+	function confirm_delete_multi(){
+		return confirm("{{$confirm_delete_multi}}");
+	}
+	function selectall(cls){
+		$("."+cls).prop("checked", true);
+		return false;
+	}
+	function unselectall(cls){
+		$("."+cls).prop("checked", false);
+		return false;
+	}
+	function details(uid) {
+		$("#user-"+uid+"-detail").toggleClass("hidden");
+		return false;
+	}
+</script>
+<style>
+	tr.details td,
+	tr.details th
+		{ border-top: 0!important; }
+</style>
+
+<div class="panel panel-default">
+	<div class="panel-body"><h1>{{$title}} - {{$page}}</h1></div>
+</div>
+
+	<form action="{{$baseurl}}/admin/users" method="post">
+		<input type='hidden' name='form_security_token' value='{{$form_security_token}}'>
+
+
+		<!--
+			**
```

Comment:
Smarty comments are done with `{{*` and `*}}`
e.g.
`{{* PENDING Users table *}}`